code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
Actually, it looks like it did still end up in the bundle, though surprisingly I didn't get any error messages about React being loaded twice :(